core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -555,7 +557,7 @@ private[log] class Cleaner(val id: Int,
 
         try {
           cleanInto(log.topicPartition, currentSegment.log, cleaned, map, retainDeletes, log.config.maxMessageSize,
-            transactionMetadata, log.activeProducersWithLastSequence, stats)
```

Comment:
We can remove this `activeProducersWithLastSequence` function now right?

Reply:
It's still used pretty heavily in test cases.